sw/device/tests/rstmgr_alert_info_test.c

```diff
@@ -695,6 +695,39 @@ static void init_expected_cause(void) {
   kExpectedInfo[kRound3]
       .alert_info.alert_cause[kTopEarlgreyAlertIdSpiHost0FatalFault] = 1;
 }
+
+// Modify kExpectedInfo for runs without rom_ext, so non-owner stages. The
+// difference is that without rom_ext we expect an otp alert in round 2.
+static void init_expected_info_for_non_rom_ext(void) {
+  if (kBootStage != kBootStageOwner) {
+    kExpectedInfo[kRound2].alert_info.class_accum_cnt[1] = 1;
+    kExpectedInfo[kRound2].alert_info.class_esc_state[1] = kCstatePhase1;
+    kExpectedInfo[kRound2]
+        .alert_info.alert_cause[kTopEarlgreyAlertIdOtpCtrlFatalBusIntegError] =
+        1;
+  }
+}
+
+void flash_ctrl_show_faults(const dif_flash_ctrl_state_t *flash_ctrl) {
```

Comment:
I like this function! It may be useful to have in flash_ctrl_testutils.

Reply:
Done.